switchbot/devices/device.py

```diff
@@ -851,9 +851,6 @@ async def _send_command(
         return result[:1] + self._decrypt(result[4:])
 
     async def _ensure_encryption_initialized(self) -> bool:
-        if self._iv is not None:
-            return True
-
```

Comment:
I think this needs a more robust fix. We probably need better synchronization to handle cases where a command is executed during or just before disconnection.